mmdet/datasets/transforms/augment_wrappers.py

```diff
@@ -94,7 +94,7 @@ class AutoAugment(RandomChoice):
     Required Keys:
 
     - img
-    - gt_bboxes (np.float32) (optional)
```

Comment:
will these changes have BC-breaking in the pipeline of up- or down-stream repo such as mmengine, mmpose, etc. ?

Reply:
To adapt the downstream repo, I add `autocase_hbox` decorator to each transform function.